force-app/lwc/signals/core.js

```diff
@@ -44,17 +50,25 @@ function $effect(fn) {
       fn();
       effectNode.error = null;
       effectNode.state = READY;
+    } catch (error) {
+      effectNode.state = ERRORED;
+      effectNode.error = error;
+      _props.errorHandler
```

Comment:
this should be probably be called onError to match the naming scheme for $signals

Reply:
I'm good with that, but I don't see there is an `onError` that currently exist at the moment

Reply:
no, but there is a onMutate